telemetry/definitions/commonDefinitions.json

```diff
@@ -116,7 +148,17 @@
         {
             "name": "aws_setCredentials",
             "description": "Select a credentials profile",
-            "metadata": [{ "type": "result" }]
+            "metadata": [{ "type": "result" }, { "type": "credentialType" }]
+        },
+        {
+            "name": "aws_setRegion",
+            "description": "A partition change occurred",
+            "metadata": [{ "type": "result" }, { "type": "credentialType" }]
```

Comment:
did you mean to use type regionId instead of credentialType? Similar for line 161

Reply:
yes - cos i'm a moron.